R/model_run.R

```diff
@@ -19,8 +19,12 @@ model_run <- function(parameters, model_version) {
     time_series,
     id_cols = "time", values_from = "value", names_from = "compartment"
   )
-  time_series$infect <- time_series$infect_asymp + time_series$infect_symp
-  time_series <- time_series[, c("infect",
+  time_series$prevalence <-
+    time_series$infect_asymp +
+    time_series$infect_symp +
+    time_series$hospitalised
+
+  time_series <- time_series[, c("prevalence",
```

Comment:
I don't recall whether this was something EPPI folks specifically asked for, but conventionally one wouldn't include the 'hospitalised' (or in need of it) in the prevalence timeseries.

We're already showing those in a different timeseries, and it can be useful for policymakers to see how peaks in hospital need lag peaks in infection.

Reply:
Yeah, they've specified they want to do this, but it's something you can push back on. Here's some of the discussion: https://teams.microsoft.com/l/message/19:meeting_MmRmODQxNWUtNzRhMC00ZDUxLWE1MjQtOGYwZDljNTA0Y2Rh@thread.v2/1727367396927?context=%7B%22contextType%22%3A%22chat%22%7D